actors.md

```diff
@@ -1597,6 +1601,10 @@ func RemoveSigner(signer Address) {
 	if !self.isSigner(signer) {
 		Fatal("given address was not a signer")
 	}
+	if len(self.Signers)-1 < self.Required {
```

Comment:
why not have an option for reducing required as well?

Reply:
Added.